packages/apps/kadena-docs/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "./node_modules/@kadena-dev/heft-rig/profiles/default/tsconfig-base.json",
+  "compilerOptions": {
+    "types": ["heft-jest", "node"],
+    "skipLibCheck": true,
+    "allowJs": true,
+    "noEmit": true,
+    "incremental": true,
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "jsx": "preserve"
+  },
+  "exclude": ["node_modules"]
```

Comment:
I think this is excluded in the `tsconfig-base`. Maybe check all props and remove duplicates.
It's preferred to have nothing here, so check if the build still works with everything removed except types, I assume